modelgrid_aliases_nuopc.xml

```diff
@@ -363,6 +363,12 @@
     <grid name="glc">ais8:gris4</grid>
     <mask>tnx1v4</mask>
   </model_grid>
+  <model_grid alias="ne30pg3_tn05">
```

Comment:
Should this have `compset="BLOM"`?
```suggestion
  <model_grid alias="ne30pg3_tn05" compset="BLOM">
```

Reply:
@gold2718 , @matsbn - I don't see why it should be made specific to BLOM, at least if we think about a version for NorESM3.5.

Reply:
@gold2718 - maybe I misunderstood, I thought you meant BLOM as opposed to BLOM%ECO, but I suppose that is not the case?

Looking through the file, I see entries wit and withaout a leading underscore, i.e. `not_compset="BLOM"` and `not_compset="_BLOM"`. Are these formats equivalent?